packages/cli/commands/import/credentials.ts

```diff
@@ -56,60 +60,104 @@ export class ImportCredentialsCommand extends Command {
 		try {
 			await Db.init();
 
+			const owner = await this.getInstanceOwner();
+			const ownerCredentialRole = await Db.collections.Role!.findOneOrFail({
+				name: 'owner',
+				scope: 'credential',
+			});
+
 			// Make sure the settings exist
 			await UserSettings.prepareUserSettings();
 			let i;
 
 			const encryptionKey = await UserSettings.getEncryptionKey();
 			if (encryptionKey === undefined) {
-				throw new Error('No encryption key got found to encrypt the credentials!');
+				throw new Error('No encryption key found to encrypt the credentials!');
 			}
 
 			if (flags.separate) {
 				const files = await glob(
 					`${flags.input.endsWith(path.sep) ? flags.input : flags.input + path.sep}*.json`,
 				);
 				for (i = 0; i < files.length; i++) {
-					// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
 					const credential = JSON.parse(fs.readFileSync(files[i], { encoding: 'utf8' }));
 
-					// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
 					if (typeof credential.data === 'object') {
 						// plain data / decrypted input. Should be encrypted first.
-						// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
 						Credentials.prototype.setData.call(credential, credential.data, encryptionKey);
 					}
 
-					// eslint-disable-next-line no-await-in-loop, @typescript-eslint/no-non-null-assertion
 					await Db.collections.Credentials!.save(credential);
+
+					const sharedCredential = new SharedCredentials();
+
+					await Db.collections.SharedCredentials!.save(
+						Object.assign(sharedCredential, {
+							user: owner,
+							credentials: credential,
+							role: ownerCredentialRole,
+						}),
+					);
```

Comment:
I know we didn't do it in commands yet but maybe we also should think about transactions?

If I am not mistaken we can create it like this:
```suggestion
					await Db.collections.SharedCredentials!.save({
							user: owner,
							credentials: credential,
							role: ownerCredentialRole,
						}),
					);
```